pkg/openstack/loadbalancer.go

```diff
@@ -621,19 +621,21 @@ func (lbaas *LbaasV2) createFullyPopulatedOctaviaLoadBalancer(name, clusterName
 		createOpts.VipAddress = loadBalancerIP
 	}
 
-	for _, port := range service.Spec.Ports {
+	for portIndex, port := range service.Spec.Ports {
 		listenerCreateOpt := lbaas.buildListenerCreateOpt(port, svcConf)
+		listenerCreateOpt.Name = cutString(fmt.Sprintf("listener_%d_%s", portIndex, name))
```

Comment:
I think using the convention is name_index instead of index_name?

Reply:
It was in previous releases, but it seems to have changed. See 

* Listener: https://github.com/kubernetes/cloud-provider-openstack/blob/85ed2f274bd96152c5603240a9a40634b7217f65/pkg/openstack/loadbalancer.go#L1908
* Pool: https://github.com/kubernetes/cloud-provider-openstack/blob/85ed2f274bd96152c5603240a9a40634b7217f65/pkg/openstack/loadbalancer.go#L1913
* Monitor: https://github.com/kubernetes/cloud-provider-openstack/blob/85ed2f274bd96152c5603240a9a40634b7217f65/pkg/openstack/loadbalancer.go#L1918